ipaserver/install/installutils.py

```diff
@@ -259,6 +291,32 @@ def record_in_hosts(ip, host_name=None, conf_file=paths.HOSTS):
 
     return None
 
+
+def order_hosts_file(host_name):
+    """Read and overwrite /etc/hosts with FQDN at beginning, issue #8018"""
+    normalize_host = ''.join(host_name)
+    hostsfile = '/etc/hosts'
+    newfile = '/tmp/hosts'
+
+    with open(hostsfile, 'r') as hostsfp, open(newfile, 'w') as outfp:
```

Comment:
In Python we don't use ``fp`` suffix for files like in C. It's not a file pointer, it's a file object. I like ``old`` and ``new`` as names.

Reply:
no problem, I changed the names, in the end they all are file descriptors. thanks.